### PR TITLE
Extend docs to include the download all feature 

### PR DESCRIPTION
**Pull request by dacog**
Extend docs to include the download all feature when filtering the documents.

### CODE REVIEW

**Comment by eikek** [comment]
Thank you very much for this contribution @dacog ! There are only some minor things regarding the screenshots. It would be awesome if you could provide a screenshot per theme (dark and light). Then you can use `imgnormal2(…)` and specify both screenshots. This would automatically pick the screenshot depending on the current theme. There are helper scripts in `website/scripts` to aid with this process a bit (but you don't need to use them, just saying).

If that is too complicated, just use `imgnormal(…)` (without the `2`) - this should work with a single screenshot as it is now.

**Comment by dacog** [comment]
Hi @eikek
Thank you for your feedback. I did not realize there was a dependency for dark and light images.

I added the dark versions (manually), but I did not set the max width to 1200px as it was not so readable. If you need them in width 1200px, just let me know.

I hope I have not missed anything now :)

File: website/site/content/docs/webapp/download-all.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
++++
+title = "Download all doocuments"
```
Comment (eikek) [inline website/site/content/docs/webapp/download-all.md:2]: A small typo: _doocuments_

**Comment by eikek** [comment]
Sorry, the ci failure is not your fault. Looks great!

**Comment by dacog** [comment]
> Sorry, the ci failure is not your fault. Looks great!

@eikek  Great! Thanks! :)

Do you still need something from my end to merge this?

**Comment by eikek** [comment]
> Do you still need something from my end to merge this?

No - all good :)

**Pull request merged**